generators/entity-client/templates/react/src/main/webapp/app/entities/react_validators.ejs

```diff
@@ -47,12 +47,12 @@ if (field.fieldValidate === true) {
     if (['Integer', 'Long', 'Float', 'Double', 'BigDecimal'].includes(fieldType)) {
         validators.push('number: { value: true, errorMessage: translate(\'entity.validation.number\') }');
     }
-    if (rules.includes('minbytes')) {
-        validators.push('minbytes: { value: ' + field.fieldValidateRulesMinbytes + ', errorMessage: translate(\'entity.validation.minbytes\', { min: ' + field.fieldValidateRulesMinbytes + ' }) }');
-    }
-    if (rules.includes('maxbytes')) {
-        validators.push('maxbytes: { value: ' + field.fieldValidateRulesMaxbytes + ', errorMessage: translate(\'entity.validation.maxbytes\', { min: ' + field.fieldValidateRulesMaxbytes + ' }) }');
-    }
+    // if (rules.includes('minbytes')) {
```

Comment:
Sorry, but I again forgot, why didi we disable this again? can you add a comment above this and other commented lines saying why we disable this so we dont forget in future

Reply:
Now that I think about it, we removed it because it's not supported well by the server. But `required` is also not supported at all by bean validation. I mean, using `@NotNull` on `byte[]` creates compilation errors on spring generated code. It might be a good idea to enable size constraints for the client only since it's already there and let the server side as it is. wdyt ?

Reply:
not sure about it, people might complain about a mismatch in validation @jdubois WDYT?

Reply:
doing it only on the client side is nice from an UI perspective, but don't protect the server at all, so that gives a false sense of security. Usually, we always take the lower common denominator when there is a mismatch with the front end and back end -> so I wouldn't do any validation here. I know it's bad, but we can't do it technically on the server side.

Reply:
Alright, I'll remove the comments too then.

Reply:
I would still prefer to leave the comments with explanation, may be we can fix it in the future